src/unix/bsd-ifaddrs.c

```diff
@@ -31,7 +31,12 @@
 #include <net/if_dl.h>
 #endif
 
-static int uv__ifaddr_exclude(struct ifaddrs *ent) {
+typedef enum {
+  UV__EXCLUDE_IFPHYS = 0,
+  UV__EXCLUDE_IFADDR = 1
+} uv__exclude_type;
```

Comment:
Not wrong but you don't need to explicitly assign values.  As well, I wouldn't use a typedef, it obscures the type.  You use it in more than one file so you can move it to src/unix/internal.h to avoid the duplication.